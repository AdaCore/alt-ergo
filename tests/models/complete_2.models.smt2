(set-option :produce-models true)
(set-logic ALL)
(declare-const x Int)
(declare-fun f (Int) Int)
; There is no constraint on `x` and `f` but we still have to produce a model
; value for them.
(check-sat)
(get-model)
