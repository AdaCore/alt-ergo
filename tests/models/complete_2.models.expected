
unknown
(
  (define-fun x () Int (as @a1 Int))
  (define-fun f ((_arg_0 Int)) Int (as @a0 Int))
)
